src/kbc_modules/offline_fs_kbc/common.rs

```diff
@@ -84,11 +97,11 @@ pub mod tests {
     #[allow(dead_code)]
     pub fn create_resources_file(resources_file_path: &Path) {
         let resources_file_content = serde_json::json!({
-          ResourceName::Policy.to_string(): base64::encode(POLICYJSON.as_bytes()),
-          ResourceName::SigstoreConfig.to_string(): base64::encode(SIGSTORECONFIG.as_bytes()),
-          ResourceName::GPGPublicKey.to_string(): base64::encode(PUBKEY.as_bytes()),
-          ResourceName::CosignVerificationKey.to_string(): base64::encode(COSIGNKEY.as_bytes()),
-          ResourceName::Credential.to_string(): base64::encode(CREDENTIAL.as_bytes()),
+            resource_path!(ResourcePath::Policy): base64::encode(POLICYJSON.as_bytes()),
```

Comment:
The value of `resource_path!(ResourcePath::Policy)` is `example.org/test-repo/security-policy/test`, right? Does `example.org` need to be removed?

Reply:
Great, you're right, sorry for the mistake